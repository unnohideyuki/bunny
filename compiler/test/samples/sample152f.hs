main = do
  print $ max LT LT
  print $ max LT EQ
  print $ max LT GT
  print $ max EQ LT
  print $ max EQ EQ
  print $ max EQ GT
  print $ max GT LT
  print $ max GT EQ
  print $ max GT GT

  print $ min LT LT
  print $ min LT EQ
  print $ min LT GT
  print $ min EQ LT
  print $ min EQ EQ
  print $ min EQ GT
  print $ min GT LT
  print $ min GT EQ
  print $ min GT GT





