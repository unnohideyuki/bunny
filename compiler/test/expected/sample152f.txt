LT
EQ
GT
EQ
EQ
GT
GT
GT
GT
LT
LT
LT
LT
EQ
EQ
LT
EQ
GT
